### DISCUSSION TITLE
Api and Secret key for structurizr LITE

### DISCUSSION THREAD

**Discussion by eon2208** [structurizr/lite#119] [Q&A] [answered]
Hi, I am trying to use Structurizr lite and create the java application to connect to the application. Unfortunately I am unable to find any information on where to find the credentials. Could you please point me in the right direction?

**Comment by simonbrowndotje** [comment, marked as answer]
> Unfortunately I am unable to find any information on where to find the credentials.

Lite is designed to work directly from the file system; see https://docs.structurizr.com/lite/usage for more.

**Reply by eon2208** [reply]
Great, thanks for the quick reply. I think it would be nice to have explicit information that the lite version cannot be used with external clients. Other than that, I really liked the tool and wish you all the best with future enhancements.

**Reply by simonbrowndotje** [reply]
> I think it would be nice to have explicit information that the lite version cannot be used with external clients.

It does say there's no workspace API on the product comparison page -> https://docs.structurizr.com/products#comparison

You're welcome to submit a documentation PR if you think this could be made clearer though.
